staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_test.go

```diff
@@ -289,6 +289,7 @@ func TestTransformationFailure(t *testing.T) {
 func TestCount(t *testing.T) {
 	ctx, cacher, terminate := testSetup(t)
 	t.Cleanup(terminate)
+	cacher.ready.set(false)
```

Comment:
I don't like it - why do you need it?

Reply:
test count get from etcd not cache

Reply:
This is by-definition tested in TestCount run from pkg/storage/etcd3/